cwms-data-api/src/main/java/cwms/cda/api/ForecastTimeseriesController.java

```diff
@@ -0,0 +1,124 @@
+                            @OpenApiContent(from = TimeSeries.class, type = Formats.JSONV2)
+                    },
+                    required = true
+            ),
+            queryParams = {
+                    @OpenApiParam(name = FORECAST_DATE_TIME, required = true, description = "Specifies the " +
+                            "forecast date time of the forecast instance to be associated with the created" +
+                            "forecast timeseries."),
+                    @OpenApiParam(name = ISSUE_DATE_TIME, required = true, description = "Specifies the " +
+                            "issue date time of the forecast instance to be associated with the created " +
+                            "forecast timeseries."),
+                    @OpenApiParam(name = OFFICE, required = true, description = "Specifies the " +
+                            "owning office of the forecast spec whose forecast instance will be " +
+                            "associated with the created forecast timeseries."),
+                    @OpenApiParam(name = SPEC_ID, required = true, description = "Specifies the " +
+                            "spec id of the forecast spec whose forecast instance will be " +
+                            "associated with the created forecast timeseries."),
+                    @OpenApiParam(name = LOCATION, required = true, description = "Specifies the " +
+                            "location of the forecast spec whose forecast instance will be" +
+                            "associated with the created forecast timeseries."),
+                    @OpenApiParam(name = TIMESERIES_ID, required = true, description = "Id of timeseries " +
+                            "that will be created.")
+            },
+            method = HttpMethod.POST,
+            path = "/forecast-timeseries",
+            tags = TAG
+    )
+    @Override
+    public void create(@NotNull Context ctx) {
+
+    }
+
+    protected DSLContext getDslContext(Context ctx) {
+        return JooqDao.getDslContext(ctx);
+    }
+
+    @OpenApi(ignore = true)
+    @Override
+    public void delete(@NotNull Context ctx, @NotNull String forecastSpecId) {
+        try (final Timer.Context ignored = markAndTime(GET_ONE)) {
+            throw new UnsupportedOperationException(NOT_SUPPORTED_YET);
+        }
+    }
+
+    @OpenApi(ignore = true)
+    @Override
+    public void getAll(@NotNull Context ctx) {
+        try (final Timer.Context ignored = markAndTime(GET_ONE)) {
+            throw new UnsupportedOperationException(NOT_SUPPORTED_YET);
+        }
+    }
+
+    @OpenApi(ignore = true)
+    @Override
+    public void getOne(@NotNull Context ctx, @NotNull String id) {
+        try (final Timer.Context ignored = markAndTime(GET_ONE)) {
+            throw new UnsupportedOperationException(NOT_SUPPORTED_YET);
+        }
+    }
+
+    @OpenApi(ignore = true)
+    @Override
+    public void update(@NotNull Context ctx, @NotNull String id) {
```

Comment:
No difference between update and create right?

Reply:
correct